test/integration/MinterUpgrade.js

```diff
@@ -0,0 +1,210 @@
+
+contract("MinterUpgrade", accounts => {
+    const NEW_INFLATION_CHANGE = new BN(1)
+
+    const transcoder1 = accounts[1]
+    const transcoder2 = accounts[2]
+    const broadcaster1 = accounts[3]
+    const broadcaster2 = accounts[4]
+
+    let controller
+    let bondingManager
+    let roundsManager
+    let token
+    let minter
+    let broker
+
+    let roundLength
+
+    const checkWithdrawalResult = async broadcaster => {
+        const unlockPeriod = await broker.unlockPeriod()
+
+        await broker.unlock({from: broadcaster})
+
+        await roundsManager.mineBlocks(unlockPeriod.mul(roundLength))
+
+        const startMinterBalance = new BN(await web3.eth.getBalance(minter.address))
+        const startInfo = await broker.getSenderInfo(broadcaster)
+
+        await broker.withdraw({from: broadcaster})
+
+        const endMinterBalance = new BN(await web3.eth.getBalance(minter.address))
+        const endInfo = await broker.getSenderInfo(broadcaster)
+
+        assert.equal(endInfo.sender.deposit.toString(), "0")
+        assert.equal(startMinterBalance.sub(endMinterBalance).toString(), startInfo.sender.deposit.toString())
+    }
+
+    before(async () => {
+        controller = await Controller.deployed()
+        await controller.unpause()
+
+        const bondingManagerAddr = await controller.getContract(contractId("BondingManager"))
+        bondingManager = await BondingManager.at(bondingManagerAddr)
+
+        const roundsManagerAddr = await controller.getContract(contractId("RoundsManager"))
+        roundsManager = await AdjustableRoundsManager.at(roundsManagerAddr)
+
+        const tokenAddr = await controller.getContract(contractId("LivepeerToken"))
+        token = await LivepeerToken.at(tokenAddr)
+
+        const minterAddr = await controller.getContract(contractId("Minter"))
+        minter = await Minter.at(minterAddr)
+
+        const brokerAddr = await controller.getContract(contractId("TicketBroker"))
+        broker = await TicketBroker.at(brokerAddr)
+
+        // Set target bonding rate to 0 so inflation decreases each round
+        await minter.setTargetBondingRate(0)
+
+        const amount = new BN(10).mul(constants.TOKEN_UNIT)
+        await token.transfer(transcoder1, amount, {from: accounts[0]})
+        await token.transfer(transcoder2, amount, {from: accounts[0]})
+
+        // Register transcoder 1
+        await token.approve(bondingManager.address, amount, {from: transcoder1})
+        await bondingManager.bond(amount, transcoder1, {from: transcoder1})
+
+        // Register transcoder 2
+        await token.approve(bondingManager.address, amount, {from: transcoder2})
+        await bondingManager.bond(amount, transcoder2, {from: transcoder2})
+
+        const deposit = new BN(web3.utils.toWei("1", "ether"))
+        await web3.eth.sendTransaction({from: accounts[0], to: broadcaster1, value: deposit})
+        await web3.eth.sendTransaction({from: accounts[0], to: broadcaster2, value: deposit})
```

Comment:
This seems rather unnecessary considering all accounts in ganache should be prefunded with 100 ETH ?

Reply:
Fixed 6ac5977